flink-state-backends/flink-statebackend-changelog/src/test/java/org/apache/flink/state/changelog/ChangelogStateBackendLoadingTest.java

```diff
@@ -225,16 +227,16 @@ public void testEnableChangelogStateBackendInStreamExecutionEnvironment() throws
         assertTrue(env.getStateBackend() instanceof MemoryStateBackend);
         assertStateBackendAndChangelogInEnvironmentAndStreamGraphAndJobGraph(
                 env, TernaryBoolean.UNDEFINED, MemoryStateBackend.class);
-        env.enableChangelogStateBackend(true);
+        //        env.enableChangelogStateBackend(true);
```

Comment:
at least add a comment to explain this?

Again, I am not sure whether removing it directly would be better.